boards/arm/ev11l78a/ev11l78a.dts

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2023, Meta Platforms, Inc. and its affiliates.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <atmel/samd20.dtsi>
+#include "ev11l78a-pinctrl.dtsi"
+
+/ {
+	model = "Microchip EV11L78A";
+	compatible = "atmel,samd20e16", "atmel,samd20";
+
+	aliases {
+		led0 = &cap_mis_led;
+	};
+
+	chosen {
+		zephyr,console = &sercom1;
+		zephyr,shell-uart = &sercom1;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		cap_mis_led: led_0 {
+			gpios = <&porta 23 GPIO_ACTIVE_HIGH>;
+			label = "CAP_MIS";
+		};
+	};
+};
+
+&cpu0 {
+	clock-frequency = <48000000>;
+};
+
+&sercom0 {
+	status = "okay";
+	compatible = "atmel,sam0-spi";
+	dipo = <0>;
+	dopo = <2>;
+	#address-cells = <1>;
+	#size-cells = <0>;
+	cs-gpios = <&porta 10 GPIO_ACTIVE_LOW>;
+
+	pinctrl-0 = <&sercom0_spi_default>;
+	pinctrl-names = "default";
+};
+
+&sercom1 {
+	status = "okay";
+	compatible = "atmel,sam0-uart";
+	current-speed = <115200>;
+	rxpo = <3>;
+	txpo = <1>;
+
+	pinctrl-0 = <&sercom1_uart_default>;
+	pinctrl-names = "default";
+};
+
```

Comment:
I see that there's a current amplifier in the schematics in PA05, you could probably set that up with a `current-sensor` node if you wanted to, see https://github.com/zephyrproject-rtos/zephyr/blob/4433ff7154ffcc0342412ae52546346e210d75b9/boards/arm/google_twinkie_v2/google_twinkie_v2.dts#L63-L68 for an example

Reply:
Sounds good!